lib/seed_migration/migration.rb

```diff
@@ -6,6 +6,20 @@ module SeedMigration
   class Migration
     include Environments
 
+    class << self
+      attr_accessor :disable_transaction
+      # Disable the transaction wrapping this migration.
+      # You can still create your own transactions even after
+      # calling #disable_transaction!
+      def disable_transaction!
+        @disable_transaction = true
+      end
+    end
+
+    def disable_transaction
+      self.class.disable_transaction
```

Comment:
```suggestion
      self.class.disable_transaction!
```

Reply:
The macro is only needed as a class method. This is to access the variable, not to change it. I renamed the variable to differentiate them.